bundles/org.eclipse.swt/Eclipse SWT Drag and Drop/win32/org/eclipse/swt/dnd/DragSource.java

```diff
@@ -491,7 +491,7 @@ private void drag(Event dragEvent) {
 	hwndDrag = 0;
 	topControl = null;
 	if (image != null) {
-		imagelist = new ImageList(SWT.NONE);
+		imagelist = new ImageList(SWT.NONE, nativeZoom);
```

Comment:
I think we need to pass DPIUtil.getZoomForAutoscaleProperty(nativeZoom) here

Reply:
Done